ci-operator/templates/openshift-azure/cluster-launch-e2e-azure-conformance.yaml

```diff
@@ -0,0 +1,236 @@
+# This template is being used for testing OpenShift on Azure. 
+# At the moment it cannot be reused to launch cluster for Origin tests
```

Comment:
is this valid now?

Reply:
I think so. But in different form :) will move